ImperatorToCk2/DataFiles/religionConversion.txt

```diff
@@ -1,4 +1,4 @@
-roman_pantheon,hellenic_pagan
+\roman_pantheon,hellenic_pagan
```

Comment:
Do you actually want the backslash here, or was that a mistake?

Reply:
Thank you for catching that, that's indeed a typo, lol